### PR TITLE
profile: add gl-ax1800 support

### PR DESCRIPTION
**Pull request by gl-zhaojianhui**
Because we have an another similar device named gl-axt1800,
so we put the common functions in a dtsi file. And we'll
submit a PR for gl-axt1800 soon.

Hardware:
* SoC: Qcom IPQ6000
* RAM: DDR3L 512MB
* Flash: 128MB Nand
* Ethernet: 5x GbE
* WLAN: 2x2 2.4GHz 600Mbps + 2x2 5GHz 1200Mbps (builtin + builtin)
* LEDS: 1x white, 1x blue
* Buttons: 1x switch, 1x reset
* USB: 1x 3.0(Type-A)
* Power: 12VDC, 1.5A

Installation:
* from qsdk to openwifi:
upgrade "openwrt-ipq807x-glinet_ax1800-squashfs-nand-factory.bin" via web ui
without keep config.
* upgrade "openwrt-ipq807x-glinet_ax1800-squashfs-sysupgrade.tar" if you are already in openwifi.

Signed-off-by: Jianhui Zhao <jianhui.zhao@gl-inet.com>

### CODE REVIEW

File: feeds/ipq807x/ipq807x/files/arch/arm64/boot/dts/qcom/qcom-ipq6018-gl-ax1800.dtsi [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+		};
+		data_7 {
+			pins = "gpio8";
+			function = "qpic_pad7";
+			drive-strength = <8>;
+			bias-pull-down;
+		};
+		qpic_pad {
+			pins = "gpio1", "gpio3", "gpio4",
+				   "gpio10", "gpio11", "gpio17";
```
[the remark this answers is not in the document]
Comment (gl-zhaojianhui) [reply feeds/ipq807x/ipq807x/files/arch/arm64/boot/dts/qcom/qcom-ipq6018-gl-ax1800.dtsi]: Fixed. Thx.

**Comment by clayface** [comment]
Have compiled and tested briefly using NetExperience controller. No issues to report. Need clarification regarding whether a dtsi and dts file for a single device is appropriate. However if the GL-AXT1800 will also be supported in OpenWiFi and make use of the dtsi then this is fine.

File: feeds/ipq807x/ipq807x/files/arch/arm64/boot/dts/qcom/qcom-ipq6018-gl-ax1800.dtsi [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+		};
+	};
+
+	usb_pins: usb_pins {
+		usb_pwr {
+			pins = "gpio0";
+			function = "gpio";
+			bias-pull-up;
+			output-high;
+		};
```
Comment (clayface) [inline feeds/ipq807x/ipq807x/files/arch/arm64/boot/dts/qcom/qcom-ipq6018-gl-ax1800.dtsi:167]: I do not seem to have USB functionality. Can you please check this?

Comment (gl-zhaojianhui) [reply feeds/ipq807x/ipq807x/files/arch/arm64/boot/dts/qcom/qcom-ipq6018-gl-ax1800.dtsi:167]: Oh. Less a kmod kmod-usb3.
Now fixed.

**Comment by blogic** [comment]
merged, Thanks !

**Pull request closed without merging**